### PR TITLE
Shifted from `reserved_nicks` file to jsonblob.com

### PR DESCRIPTION
**Pull request by arpit270896**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -2,6 +2,8 @@ SESSION_SECRET=
 OUTGOING_TOKEN=
 INCOMING_HOOK_URL=https://hooks.slack.com/services/
 API_TOKEN=xoxp-
+BLOB_URL=https://jsonblob.com/api/jsonBlob/
```
Comment (captn3m0) [inline .env.sample]: bad name for a config variable. RESERVED_NICK_URL

Also provide a sample to show how the json is formatted.

File: README.md [hunk 2]
```diff
@@ -35,6 +38,7 @@ These configuration options can either be provided via a `.env` file in developm
 1. Create a `#public` channel (could be called anything).
 2. Create an outgoing webhook that listens only on `#public`.
 3. Create an incoming webhook, and note down its URL.
+4. Create a `#public-hell` channel (could be called anything).
```
Comment (captn3m0) [inline README.md]: I'd prefer it to be called public-banned.

File: README.md [hunk 3]
```diff
@@ -42,6 +46,8 @@ Screenshots for a better understanding (outgoing and then incoming):
 
 ![Incoming Webhook](http://i.imgur.com/iCDEAok.png)
 
+![JsonBlob Sample](http://i.imgur.com/Cpo3M2i.png)
```
Comment (captn3m0) [inline README.md]: Missed this :+1:

File: README.md (hunk 2 above)
Comment (arpit270896) [inline README.md]: could be anything :stuck_out_tongue:

**Comment by arpit270896** [comment]
No. That doesn't.

**Pull request closed without merging**